src/components/noticeboard/NoticeboardCreateForm.tsx

```diff
@@ -244,6 +303,25 @@ export const NoticeboardCreateForm = ({
         />
       </Wrapper>
 
+      {showDocument && (
+        <Wrapper style={styles.noPaddingTop}>
+          <Controller
+            name="documents"
+            control={control}
+            render={({ field }) => (
+              <DocumentSelector
+                {...{ control, field }}
+                maxFileSize={documentMaxSizes.file}
```

Comment:
i always see the validation error text, but i can send the form anyways

Reply:
Problem solved